api/v2/openapi.yaml

```diff
@@ -428,6 +470,55 @@ components:
           maxItems: 100
           items:
             $ref: '#/components/schemas/Location'
+    Badge:
+      title: Type
+          description: Badges for certain actions
+          type: string
+          nullable: true
+          enum:
+            - level1
+            - level2
+            - level3
+            - level4
+            - level4
+            - level5
+            - level6
+            - level7
+            - level8
+            - level9
+            - level10
+            - shortNewRateing
+            - shortNewShopRateing
+            - shortTwoDifferentShops
+            - shortShopNeverRated
+            - middleOneRateingPerWeek
+            - middleThreeRateingsPerWeek
+            - middleDailyRatingOneWeek
+            - middleOneWeekEachDayNewShopRated
+            - longFourWeeksSameShopRated
+            - longOneMonthDailyActive
+    UserBadge:
+      title: UserBadge
+      description: Badge of a specific user
+      type: object
+      nullable: false
+      required:
+        - userId
+        - date
+        - uccupancyReportId
+      properties:
+        Type:
```

Comment:
Die einzelnen `properties` müssen in `lowerCamelCase` geschrieben werden :) Also das haben wir überall sonst auch so und das ist auch der `json`-Style wenn ich das richtig in Erinnerung habe. :D

Reply:
hab ich angepasst